tests/unit/test_routes.py

```diff
@@ -238,6 +238,17 @@ def test_create_resource(module_client, module_db, fake_auth_from_oc):
     response = create_resource(client, "invalidapikey")
     assert (response.status_code == 401)
 
+    # Invalid Resource Path
+    response = client.post('/api/v1/resources',
+        headers = {'x-apikey': apikey}
+    )
+    assert (response.status_code == 422)
+    response = client.post('/api/v1/resources',
+        json = dict(notes="Missing Required fields"),
+        headers = {'x-apikey': apikey}
+    )
+    assert (response.status_code == 422)
```

Comment:
I would like to validate the message and such, but I decided against doing that here because of #116